src/components/utils/rhf-inputs/select-inputs/input-with-popup-confirmation.js

```diff
@@ -42,6 +42,7 @@ const InputWithPopupConfirmation = ({
                 }}
             />
             <PopupConfirmationDialog
+                message={'changeTypeMessage'}
```

Comment:
I think it is not enough abstract, may be push up this property  to the parent component InputWithPopupConfirmation and make this as default value?